### PR TITLE
Test that consumption & preload of CSP-blocked resources are both reported

### PR DESCRIPTION
**Pull request by noamr**

### CODE REVIEW

File: content-security-policy/reporting/support/preload-csp-report.https.sub.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<body>
+<!-- This image will cause a CSP violation, which will trigger an immediate report -->
+<script>
+    const href = "/reporting/resources/fail.png";
+    const uid = new URLSearchParams(location.search).get('uid');
+//    const bc = new BroadcastChannel(uid);
```
Comment (annevk) [inline content-security-policy/reporting/support/preload-csp-report.https.sub.html:8]: This seems sketchy. Line that's commented out below as well.

Comment (noamr) [reply content-security-policy/reporting/support/preload-csp-report.https.sub.html:8]: Oops, was not prepared to land this yet - wanted to see that we agree on the main PR. Will prepare this one.

Comment (noamr) [reply content-security-policy/reporting/support/preload-csp-report.https.sub.html]: Done

File: content-security-policy/reporting/support/preload-csp-report.https.sub.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<body>
+<!-- This image will cause a CSP violation, which will trigger an immediate report -->
+<script>
+    const href = "/reporting/resources/fail.png";
+    const uid = new URLSearchParams(location.search).get('uid');
```
Comment (annevk) [inline content-security-policy/reporting/support/preload-csp-report.https.sub.html:7]: This is not needed then, right?

**Review by annevk — APPROVED** [review]
Looks good, modulo nit.

**Pull request merged**